sockets/client-tcp.c

```diff
@@ -48,6 +48,7 @@ int main(int argc, char ** argv)
                         break;
                 }
         }
+        message = "gr8 b8 m8, r8 8/8";
```

Comment:
find a way to remove this edit from your pull request.  Hello World is just fine...